en/components/input-group.md

```diff
@@ -131,6 +131,142 @@ The Ignite UI for Angular Input Group component can be used inside strictly type
            iframe-src="{environment:demosBaseUrl}/data-entries/typed-form" >
 </code-view>
 
+## Validation
+The following samples demonstrate how input validation could be achieved when using template-driven or reactive forms.
+
+### Template-Driven Forms
+Template-driven form validation is achieved by adding validation attributes, i.e., `required`, `minLength`, etc., to the `input` element.
+
+```html
+<igx-input-group>
```

Comment:
Include the wrapping form element in each code snippet, together with the submit button.